File: ai/alfredo01.py
# coding: utf-8
import os

os.system("cls" if "windows" in os.name.lower() else "clear")

resposta = raw_input("\nOlá, eu sou Alfredo. Em que posso ajudá-lo?\n")

print "\nVocê disse: " + resposta

while (True):
    resposta = raw_input("\nPosso ajudá-lo em mais alguma coisa?\n")
    if ("não" in resposta.lower()):
        break

print "\nObrigado e até logo\n\n"
 

File: ai/alfredo01.pl
:- initialization(main).

main :-
    put(27),write('[2J'),ttyflush,
    write('Olá, eu sou Alfredo, em que posso ajudar?'),nl,
    get_code(C),
    write('Você disse:'),
    format('~c',C),nl, 
    halt.
